src/xdg.c

```diff
@@ -513,12 +527,20 @@ xdg_toplevel_view_configure(struct view *view, struct wlr_box geo)
 static void
 xdg_toplevel_view_close(struct view *view)
 {
+	if (!xdg_toplevel_from_view(view)->base->initialized) {
```

Comment:
`xdg_toplevel.close` is not a part of a configure sequence so this check isn't strictly required.

Reply:
Thanks, removed.